cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case1.prm

```diff
@@ -8,6 +8,9 @@
 # between the subducting and overriding plate, so about
 # 100 MPI processes are required.
 
+# We included a shared library for using the trench location postprocessor.
+set Additional shared libraries            = /home/bbkyingy/software/aspect_dealii_9.4.0/aspect/cookbooks/kinematically_driven_subduction_2d/libsubduction_plate_cooling.so
```

Comment:
```suggestion
set Additional shared libraries            = $ASPECT_SOURCE_DIR/cookbooks/kinematically_driven_subduction_2d/libsubduction_plate_cooling.so
```
This way the path to the shared library is not an absolute path on _your_ computer, but a relative path that can always be found.